dot-atom/styles.less

```diff
@@ -4,22 +4,23 @@
  * This stylesheet is loaded when Atom starts up and is reloaded automatically
  * when it is changed.
  *
- * If you are unfamiliar with LESS, you can read more about it here:
- * http://www.lesscss.org
+ * Uncomment the examples below to try it out. Or add your own CSS or Less.
```

Comment:
Refers to "reloaded automatically", but I shuffled it around. Now it's under the "Examples".

Still not sure about `(To see them, uncomment and save)`.